Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoIse.yaml

```diff
@@ -0,0 +1,125 @@
+    //Dvc specific fields
+    | extend 
+          DvcIpAddr = extract(@"(d{1,3}\.\d{1.3}\.\d{1,3}\.\d{1,3})", 1, Computer)
+        , DvcHostname = coalesce(tostring(extract(@"NetworkDeviceName=(.*?),", 1, SyslogMessage)), Computer, HostName)
+    //Authentication specific fields
+        , LogonProtocol = extract(@"Protocol=(.*?),", 1, SyslogMessage)
+    //Target specific fields
+        , TargetIpAddr = extract(@"DestinationIPAddress=(.*?),", 1, SyslogMessage)
+        , TargetPortNumber = toint(extract(@"DestinationPort=(.*?),", 1, SyslogMessage))
+        , TargetMacAddr = tostring(extract(@"Called-Station-ID=([a-fA-F0-9\-:]{17}),", 1, SyslogMessage))
+        , TargetUsername = coalesce(tostring(extract(@"User-Name=(.*?),", 1, SyslogMessage)), tostring(extract(@"UserName=(.*?),", 1, SyslogMessage)), tostring(extract(@"User=(.*?),", 1, SyslogMessage)))
+        , TargetUsernameType = case(EventMessage has_any ("host", "machine"), "Machine",
+                                    EventMessage has_any ("administrator", "root"), "Admin", "Regular")
+        , TargetSessionId = tostring(extract(@"SessionID=([a-fA-F0-9]+),", 1, SyslogMessage))
+    //Src specific fields
+    | extend 
+        SrcIpAddr = case(SyslogMessage has "Remote-Address", extract(@"Remote-Address=(.*?),", 1, SyslogMessage),
+                          SyslogMessage matches regex @"Caller-Station-ID=\d{1,3}\.\d{1.3}\.\d{1,3}\.\d{1,3}", extract(@"Caller-Station-ID=(\d{1,3}\.\d{1.3}\.\d{1,3}\.\d{1,3})", 1, SyslogMessage), extract(@"Device IP Address=(.*?),", 1, SyslogMessage))
+        , SrcPortNumber = toint(extract(@"Device Port=(\d{1,5})", 1, SyslogMessage))
+        , SrcMacAddr = coalesce(replace_string(tostring(extract(@"Caller-Station-ID=([a-fA-F0-9\-:]{17})", 1, SyslogMessage)), "-", ":"), tostring(extract(@"device-mac=(.*?),", 1, SyslogMessage)))
+    | extend 
+        SrcHostname = case(SyslogMessage matches regex @"Caller-Station-ID=\d{1,3}\.\d{1.3}\.\d{1,3}\.\d{1,3}", "",
+                        isnotempty(SrcMacAddr), "",
+                        tostring(extract(@"Caller-Station-ID=([a-fA-F0-9\-]{17})", 1, SyslogMessage)))
+  // ********************** <Postfilterring> **********************************
+    | where (targetusername_has=='*' or TargetUsername has targetusername_has)
+  // ********************** </Postfilterring> *********************************     
+  // ************************* <Aliases> **********************
+    | extend 
+        Dvc = coalesce(DvcIpAddr, DvcHostname)
+        , IpAddr = SrcIpAddr
+        , Dst = TargetIpAddr
+        , Src = SrcIpAddr
+        , User = TargetUsername
+  // ************************* </Aliases> ******************** 
+    | project-away TenantId, SourceSystem, MG, Computer, EventTime, Facility, HostName, SeverityLevel, SyslogMessage, HostIP, ProcessName, ProcessID, _ResourceId, FailureReason
+  };
+  CiscoISEAuthParser(starttime, endtime, targetusername_has, disabled)
```

Comment:
use the format CiscoISEAuthParser(starttime=starttime, endtime=endtime, targetusername_has=targetusername_has, disabled=disabled)

Reply:
fixed